cpp/src/lance/format/schema.cc

```diff
@@ -345,6 +346,7 @@ std::shared_ptr<Field> Field::Copy(bool include_children) const {
   new_field->logical_type_ = logical_type_;
   new_field->extension_name_ = extension_name_;
   new_field->encoding_ = encoding_;
+  new_field->dictionary_ = dictionary_;
```

Comment:
this is for dictionary arrays? or is this some other dictionary?

Reply:
Yes, this is the dictionary value array (i.e., strings), do not contain indices.